bin/reth/src/utils.rs

```diff
@@ -90,6 +91,11 @@ impl<'a, DB: Database> DbTool<'a, DB> {
         self.db.view(|tx| tx.get::<T>(key))?.map_err(|e| eyre::eyre!(e))
     }
 
+    /// Grabs the content of the table for the given key and subkey
+    pub fn get_dup<T: DupSort>(&mut self, key: T::Key, subkey: T::SubKey) -> Result<Option<T::Value>> {
+        self.db.view(|tx| tx.cursor_dup_read::<T>().unwrap().seek_by_key_subkey(key, subkey)).unwrap().map_err(|e| eyre::eyre!(e))
```

Comment:
@rkrasiuk is this the right way? IIRC we cannot do this fully generically and need to use the structs we've defined for the subkeys by table?

Reply:
pinging @rkrasiuk I'm back on the issue. `seek_by_key_subkey` and `cursor_dup_read` forces me to pass the Subkey here, so it's impossible having the same macro for both types of tables (`Table` and `DupSort`), even with new changes from https://github.com/paradigmxyz/reth/pull/2935

So if this call with `tx.cursor_dup_read::<T>().unwrap().seek_by_key_subkey(key, subkey)` is the correct way of pulling data, we would need to change macro from https://github.com/paradigmxyz/reth/pull/2935 and create 2 separate macros for "normal" and "DupSort" tables. OR we create a generic method for DupSort tables but keep the `view` function only having `fn view<T: Table>` without DupSort.


Let me know your thoughts.